### DISCUSSION TITLE
Is there any way to add an indication that a user has entered into a "virtualenv".

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [python-poetry/poetry#5269] [Q&A] [answered]
Hi 👋 Sorry if this has already been answered

Basically when I run `poetry shell`, it spawns a new virtual environment and uses it. But there is no indication in the command prompt that we are inside the created virtual environment.

If I use `virtualenv` to create an virtual environment and use `activate.bat` script, I get [Notice the `(venv)` in brackets]. But `poetry shell` doesn't give a similar indication. Any way to do that ?

![image](https://user-images.githubusercontent.com/34304644/156571654-e196ba7d-07eb-49ab-8764-b6cd68ae207f.png)

**Comment by abn** [comment, marked as answer]
On windows, in current releases of poetry, there is a bug (#4996) that does not activate the environment correctly. This was resolved with #5053. 

The next pre-release for 1.2 should have this fix.

**Reply by UltraInstinct05** [reply]
Thanks for the info !
